getting-started/kubernetes/helm.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Installing Calico with Helm on Kubernetes
+description: Install Calico on a Kubernetes cluster using Helm 3.
+canonical_url: '/getting-started/kubernetes/helm'
+---
+
+### Big picture
+
+This installs Calico on a Kubernetes cluster using Helm 3.
+
+### Value
+
+Helm charts are a way to package up an application for Kubernetes.  This can be thought of as similar to the way `apt` or `yum` packages applications for operating systems.  
+
+Helm is also used by tools like ArgoCD to manage applications in a cluster, taking care of install, upgrade (and rollback if needed), etc. 
+
+### Before you begin
+
```

Comment:
**Required**

- Helm 3 is installed on the cluster.
- Kubernetes cluster meets these requirements:
  - Kubernetes installed, but *without* a CNI plugin **OR** a cluster running a compatible CNI for Calico to run in policy-only mode 
  - x86-64, arm64, ppc64le, or s390x processors
  - RedHat Enterprise Linux 7.x+, CentOS 7.x+, Ubuntu 16.04+, or Debian 9.x+

- `kubeconfig` is configured to work with your cluster (check by running `kubectl get nodes`)

- {{site.prodname}} can manage `cali` and `tunl` interfaces on the hosts.
  If NetworkManager is present on the hosts, refer to
  [Configure NetworkManager](../../maintenance/troubleshoot/troubleshooting#configure-networkmanager).